src/app/+my-dspace-page/my-dspace-new-submission/my-dspace-new-submission.component.ts

```diff
@@ -118,6 +125,28 @@ export class MyDSpaceNewSubmissionComponent implements OnDestroy, OnInit {
     this.modalService.open(CreateItemParentSelectorComponent);
   }
 
+  /**
+   * Method invoked after all file are loaded from upload plugin
+   */
+  afterFileLoaded(items) {
+    const uploader = this.uploaderComponent.uploader;
+    if (hasValue(items) && items.length > 1) {
+      this.notificationsService.error(null, this.translate.get('mydspace.upload.upload-failed-moreonefile'));
```

Comment:
I had thought we decided that users should be allowed to upload more than one file from the MyDSpace page. Per the ticket https://jira.lyrasis.org/browse/DS-4514?focusedCommentId=64782&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-64782 we said...

> If multiple files are uploaded at once, they are all attached to a single WorkspaceItem, and then each is checked for bibliographic information. The first bibliographic info found is used to populate the submission form (and any other bibliographic files are thereafter ignored). This behavior should be documented, as it may mean users will want to first upload a single bibliographic file of choice to begin the submission, and then upload other related files in the submission form. (Obviously this is not ideal behavior, but it keeps things simple for now.)

Was there a reason for this change in behavior?

Reply:
Closing this request.  It's a limitation of the library we are using (see new ticket #820) so this PR is doing the correct thing in displaying a useful error message.